projects/balanced/index.js

```diff
@@ -0,0 +1,100 @@
+const retry = require("async-retry");
+const axios = require("axios");
+const BigNumber = require("bignumber.js");
+
+const icxApiEndpoint = 'https://ctz.solidwallet.io/api/v3'
+const balancedDexContract = 'cxa0af3165c08318e988cb30993b3048335b94af6c'
+const balancedLoansContract = 'cx66d4d90f5f113eba575bf793570135f9b10cece1'
+const bandOracleContract = 'cx087b4164a87fdfb7b714f3bafe9dfb050fd6b132'
+const bnusdContract = 'cx88fd7df7ddff82f7cc735c871dc519838cb235bb'
+const sicxContract = 'cx2609b924e33ef00b648a409245c7ea394c467824'
+
+async function icxCall(address, method, params) {
+    try {
```

Comment:
Is it possible to remove the try/catch? I'd rather have it fail hard and be alerted of it rather than have it fail partially and get incorrect data?

Reply:
I made a new commit and removed the try/catch block. I tested it with an incorrect endpoint and it hard failed. Let me know if it looks okay.

Reply:
Thanks, it was just a very minor thing